api/components/v1alpha1/trustyai_types.go

```diff
@@ -54,8 +54,26 @@ type TrustyAISpec struct {
 	TrustyAICommonSpec `json:",inline"`
 }
 
+// TrustyAIEvalSpec defines evaluation configuration for TrustyAI
+type TrustyAIEvalSpec struct {
+	// LMEval configuration for model evaluations
+	LMEval TrustyAILMEvalSpec `json:"lmeval,omitempty"`
+}
+
+// TrustyAILMEvalSpec defines configuration for LMEval evaluations
+type TrustyAILMEvalSpec struct {
+	// AllowCodeExecution controls whether code execution is allowed during evaluations
+	// +kubebuilder:default=false
+	AllowCodeExecution *bool `json:"allowCodeExecution,omitempty"`
```

Comment:
since the field has a default value, is is required to define it as a pointer ?

Reply:
@lburgazzoli good point, removed pointer.